### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by gotroyb127** [cdown/clipmenu#128]
I was getting errors like:
````
/usr/bin/clipmenud: line 152: ((: > CM_MAX_CLIPS_THRESH : syntax error: expected binary operator (the error is "> CM_MAX_CLIPS_THRESH ")
/usr/bin/clipmenud: γραμμή 152: /run/user/1000/clipmenu.6.username//line_cache: No such file or directory.
````
so i just removed the trailing `/` from line 16 (`cache_dir=$CM_DIR/clipmenu.$major_version.$USER/`), so now it looks like
`cache_dir=$CM_DIR/clipmenu.$major_version.$USER
`

It seems that that slash was creating the error. I just commented so that you know.

**Comment by cdown** [comment]
That's certainly not the reason :-)

POSIXly, path1//path2 and path1/path2 are the same thing, so this doesn't make any difference. More likely is that the line_cache was unexpectedly deleted.

What version of clipmenu was this with?

**Comment by cdown** [comment]
Looks like this should have already been resolved by 2677932c560fa54c6c22fa706d01c0d17303f1a8, it's just not in a released version yet.

There should be a release in the not too distant future.

**Comment by gotroyb127** [comment]
It seems that you are right. It tricked me though as it seemed like it stopped throwing errors (although actually it didn't). 

I am using version 6.0.1-1 according to pacman (I have install it from the official ArchLinux repos).

I patched the 2677932 as you said and I am currently getting no errors.

Thank you.

**Comment by cdown** [comment]
Good news is 6.1.0 is now out :-)
